### PR TITLE
fix stock-knative runner.

### PR DESCRIPTION
**Pull request by shyamjesal**
Signed-off-by: Shyam Jesal <s.jesalpura@gmail.com>

### CODE REVIEW

File: runner/scripts/setup-runner.sh [hunk 1]
```diff
@@ -26,6 +26,7 @@ set -e
 
 # Install base- and setup-dependencies
 
+sed -i -re 's/([a-z]{2}\.)?archive.ubuntu.com|security.ubuntu.com/old-releases.ubuntu.com/g' /etc/apt/sources.list
```
Comment (ustiugov) [inline runner/scripts/setup-runner.sh:29]: please add a comment

File: runner/scripts/install-konk.sh [hunk 2]
```diff
@@ -22,7 +22,6 @@ KNATIVE_DOMAIN=$INGRESS_HOST.nip.io
 kubectl patch configmap -n knative-serving config-domain -p "{\"data\": {\"$KNATIVE_DOMAIN\": \"\"}}"
 
 echo -e "🔥 Installing Knative Eventing... \033[0m"
```
Comment (ustiugov) [inline runner/scripts/install-konk.sh:24]:
```suggestion
```

**Review by ustiugov — CHANGES_REQUESTED** [review]

**Review by dhschall — APPROVED** [review]
Looks good to me!
Thanks for resolving

**Comment by dhschall** [comment]
@ustiugov I guess this is resolved?

**Pull request merged**